modules/installation-configuration-parameters.adoc

```diff
@@ -1086,7 +1086,7 @@ Additional Azure configuration parameters are described in the following table:
 |String, for example `production_disk_encryption_set`.
 
 |`controlPlane.platform.azure.osDisk.diskEncryptionSet.subscriptionId`
-|Optional. The ID of a disk encryption set in another Azure subscription. This secondary disk encryption set will be used to encrypt control plane machines. By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.
+|The ID of a disk encryption set in another Azure subscription. This secondary disk encryption set will be used to encrypt control plane machines. By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.
```

Comment:
Same as compute, here also needs to remove the statement of "`By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts."`

Reply:
It sounds not accurate for `The ID of a disk encryption set in another Azure subscription`.
I checked more from explain of installer.
```
$ ./openshift-install explain installconfig.platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.subscriptionId
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  SubscriptionID defines the Azure subscription the disk encryption set is in.
```
Suggest to reword `The ID of a disk encryption set in another Azure subscription`, how about change to `The ID of the Azure subscription the disk encryption set is in`?

same update also need to be applied on `compute.platform.azure.osDisk.diskEncryptionSet.subscriptionId`

Reply:
@jinyunma I updated the PR and reworded the `  SubscriptionID defines the Azure subscription the disk encryption set is in.` sentence. can you PTAL?

Reply:
thanks for addressing this.

Add @jhixson74  to review
The doc changes the description of platform.azure.osDisk.diskEncryptionSet.subscriptionId
`The ID of a disk encryption set in another Azure subscription.`
=> 
`Defines the Azure subscription of the disk encryption set, where the disk encryption set resides`

@jhixson74 Do you think is it reasonable? thanks.